pyanaconda/ui/gui/spokes/custom_storage.py

```diff
@@ -1281,14 +1281,10 @@ def on_modify_container_clicked(self, button):
             return
 
         # Update the UI.
-        idx = None
+        # we're looking for the original vg name
+        idx = next((i for i, data in enumerate(self._containerStore) if data[0] == container_name), None)
 
-        for idx, data in enumerate(self._containerStore):
-            # we're looking for the original vg name
-            if data[0] == container_name:
-                break
-
-        if idx:
+        if idx is not None:
```

Comment:
Again doesn't look to me like improvement.

Reply:
So keep this rule disabled ? 
Or only this occurrence?

I dont have preference.